wmn-data.json

```diff
@@ -6560,5 +6560,15 @@
         "known" : ["carlito", "taylor"],
         "cat" : "social"
       }
+      {
+        "name" : "Letterboxd",
+        "uri_check" : "https://letterboxd.com/{account}",

```

Comment:
Can you add a `/` to the end of the `uri_check` as I'm getting redirected without it. So it should be `https://letterboxd.com/{account}/`